core/frontend/src/render/webgl/PlanarTextureProjection.ts

```diff
@@ -14,7 +14,8 @@ import {
 import { Frustum, FrustumPlanes, Npc, RenderMode } from "@bentley/imodeljs-common";
 import { ApproximateTerrainHeights } from "../../ApproximateTerrainHeights";
 import { SceneContext } from "../../ViewContext";
-import { Tile, TileTreeReference } from "../../tile/internal";
+import { Tile } from "../../tile/Tile";

```

Comment:
no, this needs to import from tile/internal.

Reply:
I will rerun and make sure the linter doesn't autofix this... it's configured not to, I probably just messed this up myself while merging with master.

Reply:
yes the rule doesn't report or try to fix this, I accidentally preserved it while merging. pushed a fix